workers/unity/Packages/io.improbable.gdk.tools/GdkToolsConfigurationWindow.cs

```diff
@@ -160,5 +164,26 @@ private void DrawCodeGenerationOptions()
 
             EditorGUIUtility.labelWidth = previousWidth;
         }
+
+        private void DrawCustomSnapshotDir()
+        {
+            GUILayout.Label(CustomSnapshotPathLabel, EditorStyles.boldLabel);
+            GUILayout.Space(EditorGUIUtility.standardVerticalSpacing);
+
+            using (new EditorGUILayout.HorizontalScope())
+            {
+                EditorGUI.BeginDisabledGroup(true);
```

Comment:
You can use a `DisabledScope` with a `using` block instead of the Begin/End methods - https://docs.unity3d.com/ScriptReference/EditorGUI.DisabledScope.html

Makes it less error prone imo